examples/datasets/dataset_partitioners.py

```diff
@@ -2,14 +2,15 @@
 from abc import ABC, abstractmethod
```

Comment:
The changes in this file simply relate to moving from the previous news classification dataset to the ag news dataset.